package/Dockerfile

```diff
@@ -1,3 +1,14 @@
-FROM alpine:3.14.2
+FROM registry.suse.com/bci/bci-micro:15.3.16.3
+
+ARG user=webhookreceiver
+
+RUN echo "$user:x:1000:1000::/home/$user:/bin/bash" >> /etc/passwd && \
+    echo "$user:x:1000:" >> /etc/group && \
+    mkdir /home/$user && \
+    chown -R $user:$user /home/$user
+
 COPY bin/webhook-receiver /webhook-receiver
+
```

Comment:
You might want to also set the current working directory to the `/home/$user` directory and copy/run the file out of there. 

I mention this because you went through the effort of making a home directory for the user - would be a shame not to use it.

Reply:
I would still leave the binary in `/bin/webhook-receiver`. 

I believe it's better to just remove the `mkdir $home` command. It 2 lines less in the file and should not cause any harm.

WDYT?

Reply:
Thanks for the extra comment. So let's leave as it is right now. It will not hurt having the home folder but not using it.